example/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@
 cmake_minimum_required(VERSION 3.1)
 project(spdlog_examples CXX)
 
+include(../cmake/utils.cmake)
```

Comment:
why this i needed?

Reply:
Otherwise `spdlog_enable_warnings` isn't recognized when using `example/CMakeLists.txt ` as a stand-alone CMakefile